.github/workflows/docker.yml

```diff
@@ -18,6 +18,7 @@ on:
           - '**.toml'
           - '/docker/**'
           - '.github/workflows/docker.yml'
+    workflow_dispatch:
```

Comment:
That's all that is needed?
If so, why only for the docker workflow?
Shouldn't it be added to all workflows?

Reply:
It sure should, I could go into a more granular approach, but the defaults are good for now. That said, I should update the other workflows as well :)

Reply:
Awesome.
Ping me when you’re done !

Reply:
I just did! 😄